homeassistant/components/tradfri/fan.py

```diff
@@ -171,6 +172,7 @@ async def async_turn_off(self, **kwargs: Any) -> None:
     def _refresh(self, device: Command, write_ha: bool = True) -> None:
         """Refresh the purifier data."""
         # Caching of air purifier control and purifier object
+        self._device = device
```

Comment:
We don't need this. It's done here already:
https://github.com/home-assistant/core/blob/dee843bf6e5ca84a94f336a239f6a6138c4c28e6/homeassistant/components/tradfri/base_class.py#L106